src/helics/network/CommsInterface.hpp

```diff
@@ -130,8 +134,9 @@ class CommsInterface {
     const bool singleThread{false};  //!< specify that the interface should operate a single thread
 
   protected:
-    bool serverMode = true;  //!< some comms have a server mode and non-server mode
-    bool autoBroker = false;  //!< the broker should be automatically generated if needed
+    bool clientMode{false};  //!< specify that the comms should assume we have a broker
+    bool serverMode{true};  //!< some comms have a server mode and non-server mode
```

Comment:
How much would break if both clientMode and serverMode are set to the same value?

Reply:
They control different things, so nothing would break.    The connectionPort setting if used would get set to to use the broker,  which might be slightly unexpected, but there are also a few cases where this actually might be the case as in the tcp_ss server.  The clientMode controls the direction the "connectionPort" and "connectionAddress"  the serverMode controls whether it acts as a network server.    I suppose this isn't as clear as it could be